test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -1147,7 +1147,19 @@ func fixJSONOutput(b *testing.B) {
 // You can pass your own scheduler plugins via outOfTreePluginRegistry.
 // Also, you may want to put your plugins in PluginNames variable in this package
 // to collect metrics for them.
-func RunBenchmarkPerfScheduling(b *testing.B, configFile string, topicName string, outOfTreePluginRegistry frameworkruntime.Registry) {
+func RunBenchmarkPerfScheduling(b *testing.B, configFile string, topicName string, outOfTreePluginRegistry frameworkruntime.Registry, options ...SchedulerPerfOption) {
+	opts := &schedulerPerfOptions{
+		prepareFn: nil,
+	}
+
+	for _, option := range options {
+		option(opts)
+	}
+
+	runBenchmarkPerSchedulingImpl(b, configFile, topicName, outOfTreePluginRegistry, opts)
+}
+
+func runBenchmarkPerSchedulingImpl(b *testing.B, configFile string, topicName string, outOfTreePluginRegistry frameworkruntime.Registry, options *schedulerPerfOptions) {
```

Comment:
why split into two functions?

Reply:
To clearly separate converting `SchedulerPerfOption` to `schedulerPerfOptions` from the actual logic. But it seems like there's no need, so I'll merge them.